src/test/run-pass/cfg/cfg-family.rs

```diff
@@ -10,3 +8,7 @@ pub fn main() {
 #[cfg(unix)]
 pub fn main() {
 }
+
+#[cfg(not(any(windows, unix)))]
```

Comment:
Doesn't this miss the point of test? (Same for target_family test.)

Reply:
What's your suggestion for making the test suite pass on non-windows non-unix platforms? It seems silly to me to add an `ignore` statement for every single such platform that may be added to Rust.

Reply:
what exactly *is* the point of this test? I guess it's to test that you have *either* windows *or* unix (except when you don't, hence the ignores?)

In that case, perhaps we want to add some kind of "meta ignore" statement instead?

Reply:
But, honestly, adding some ignore statements doesn't seem terrible for the time being. =) How often do we add such platforms anyway? And it's only two more tests.

Reply:
> what exactly *is* the point of this test?

I think it's to test that `cfg(unix)` works, as in, that the compiler correctly sets the configuration based on the target specification.